R/deployApp.R

```diff
@@ -324,29 +331,28 @@ deployApp <- function(appDir = getwd(),
     bundle <- application$deployment$bundle
   }
 
-  if (length(bundle$id) > 0 && nzchar(bundle$id)) {
```

Comment:
Do you think that we still log enough (early) information about the deployment target? Output from rsconnect is often attached to support tickets, and breadcrumbs like these are occasionally helpful.

Reply:
Yeah, let me try again.